src/python/pants/BUILD

```diff
@@ -25,7 +25,6 @@ python_distribution(
   provides=setup_py(
     name='pantsbuild.pants',
     description='A scalable build tool for large, complex, heterogeneous repos.',
-    namespace_packages=['pants', 'pants.backend'],
```

Comment:
I'm not sure why we were declaring `pants.backend`. That is inferred due to `pants`, afaict.